python/vosk/__init__.py

```diff
@@ -1,8 +1,17 @@
 import os
 import sys
 
+from requests import get
+from urllib.request import urlretrieve
+from zipfile import ZipFile
+from re import match
+from pathlib import Path
 from .vosk_cffi import ffi as _ffi
 
+MODEL_PRE_PATH = 'https://alphacephei.com/vosk/models/'
+MODEL_LIST_URL = MODEL_PRE_PATH + 'model-list.json'
+MODELS_HOME_DIR = Path.home() / '.cache' / 'vosk'
```

Comment:
Add 3 folders to look for the models:

/usr/share/vosk
$HOME/.cache/vosk
os.env('VOSK_MODEL_PATH')